process/readme_guide.md

```diff
@@ -0,0 +1,40 @@
+# Writing a README
+All code repos should include a README that gives a summary of what the code does and provides necessary instructions for setup and use.
+
+## Formatting
+A README file should generally be written in [github-flavored Markdown](https://help.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github), unless it will be presented somewhere other than GitHub. Markdown files can be previewed in editors such as [VS code](https://code.visualstudio.com/docs/languages/markdown) and [Macdown](https://macdown.uranusjr.com/), or by searching for an online Markdown editor.
+
+Information should be split into sections that make the instructions easy to follow. Use formatting options correctly to ensure accessibility. Header levels should appropriately divide sections and subsections, and there should be a single `h1` top-level header. In general, keep in mind guidelines and best practices for writing [accesible content](https://developers.google.com/style/accessibility).
```

Comment:
Change "easy" to "clear". 

https://dev.to/meeshkan/how-to-remove-condescending-language-from-documentation-4a5p

Reply:
[accesible content] typo in accessible.